lib/octokit/client/repositories.rb

```diff
@@ -696,6 +715,30 @@ def update_subscription(repo, options = {})
       def delete_subscription(repo, options = {})
         boolean_from_response :delete, "#{Repository.path repo}/subscription", options
       end
+
+      # Check if a repository is a template repository
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Boolean] True if repository is a template repository, false otherwise.
+      def template_repository?(repo)
+        options = ensure_api_media_type(:template_repositories, {})
+        response = get Repository.path(repo), options
+        response.is_template?
+      end
+      alias :template_repo? :template_repository?
```

Comment:
I'm kind of against a helper method like this being in the client.

The only time we support `def *?` methods is when we are expecting a `204` response.

https://github.com/octokit/octokit.rb/blob/9bc8cd25c335cadbc898e11976a32aa8ce77ec8c/lib/octokit/client/organizations.rb#L193-L211

Reply:
Makes sense. I added this to avoid updating the existing `get repository` method with the preview key.